hooks/terraform_docs.sh

```
@@ -231,8 +233,13 @@ function terraform_docs {
       have_marker=$(grep -o "$insertion_marker_begin" "$output_file") || unset have_marker
       [[ ! $have_marker ]] && popd > /dev/null && continue
     fi
+    if [[ $had_config_flag == true ]]; then
+    # shellcheck disable=SC2086
+    terraform-docs --output-mode="$output_mode" --output-file="$output_file" $tf_docs_formatter --config="$config_file" $args ./ > /dev/null
+    else
     # shellcheck disable=SC2086
-    terraform-docs --output-mode="$output_mode" --output-file="$output_file" $tf_docs_formatter $args ./ > /dev/null
+    terraform-docs --output-mode="$output_mode" --output-file="$output_file" $tf_docs_formatter $config_arg $args ./ > /dev/null
```

Shellcheck (0.10.0)

[warning] config_arg is referenced but not assigned.

(SC2154)